### ISSUE TITLE
Allowing a request dose not work at tv.naver.com

### ISSUE THREAD

**Issue by piquark6046** [AdguardTeam/AdguardForiOS#2315]
### Please answer the following questions for yourself before submitting an issue

- [X] Filters were updated before reproducing an issue
- [X] I checked the [knowledge base](https://adguard.com/kb/) and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard version

4.5.2

### Environment

- OS: 17.4.1
- Device: iPhone 15

### Ad Blocking

AdGuard Base filter, AdGuard Mobile Ads filter

### Privacy

AdGuard Tracking Protection filter

### Language-specific

List-KR

### Advanced protection for Safari

- [X] yes, I do

### Which DNS server do you use?

AdGuard DNS Non-filtering

### DNS protocol

DNS-over-QUIC

### DNS filtering

- [ ] yes, I do

### DNS implementation

AdGuard

### Tunnel mode

Split-Tunnel

### Low-level settings

Bootstrap server:
Fallback server:
Blocking mode, etc:

### Issue Details

Steps to reproduce:
1. Configure your AdGuard for iOS as mentioned in this issue.
2. Connect to a AdGuard VPN server located in South Korea.
3. Visit the provided URL: https://tv.naver.com/v/49487950

### Expected Behavior

The anti-adblock message does not appear because `@@||ssl.pstatic.net/tveta/libs/ssp-video/prod/ssp.web.sdk_$script,domain=tv.naver.com` existing List-KR filter allows a related request. 

### Actual Behavior

`@@||ssl.pstatic.net/tveta/libs/ssp-video/prod/ssp.web.sdk_$script,domain=tv.naver.com` rule does not work.
Therefore, you can see the anti-adblock message.

### Screenshots

<details><summary>Screenshot 1</summary>

  ![Screenshot 1](https://cdn.adguard.info/sitereports/bo1f6g88y348sk44osccccswg4c8ckgpnitatf6qog.png?nc=1)

</details>

### Additional Information

https://github.com/AdguardTeam/AdguardFilters/issues/176493

**Comment by maxikuzmin** [comment]
>Connect to a AdGuard VPN server located in South Korea.

@piquark6046 could you please clarify what you mean by that?

**Comment by piquark6046** [comment]
To play the video.

**Comment by maxikuzmin** [comment]
@piquark6046 as far as I can see you only attached a screenshot?

**Comment by piquark6046** [comment]
Updated the issue comment.
I think that using only the provided URL to reproduce is best for investigating.

**Comment by maxikuzmin** [comment]
@piquark6046 could you clarify if I understood correctly that you need to be in South Korea to reproduce the problem?

**Comment by piquark6046** [comment]
> could you clarify if I understood correctly that you need to be in South Korea to reproduce the problem?

No. you don't need to live in South Korea to reproduce the issue.
If you connect to a VPN server located in non South Korea, you will face a region restriction  error.

<details><summary>Example (a ProtonVPN Japan server)</summary>
<p>

![image](https://github.com/AdguardTeam/AdguardForiOS/assets/98787049/13cfa92e-bf54-4801-b446-709d77d82642)

</p>
</details>

**Comment by maxikuzmin** [comment]
@piquark6046 I see, will fix in the next updates 
Thank you!

**Comment by piquark6046** [comment]
@maxikuzmin Can take a look https://github.com/List-KR/List-KR/issues/913?
The issue can be reproduced with only List-KR filter.

**Comment by zloyden** [comment]
Found a similar problem, but on other site https://github.com/AdguardTeam/AdguardFilters/issues/179893.
Seems to be that the exception rule does not apply there, but I don't get the same on other platforms.
```adblock
@@||yads.c.yimg.jp/js/yads-async.js$domain=kobe-np.co.jp|yahoo.co.jp
```
